recipes/anypytools/bld.bat

```diff
@@ -0,0 +1,6 @@
+"%PYTHON%" setup.py install 
+
+cd "%PREFIX%"
+mkdir AnyPyToolsTutorial
```

Comment:
Yes. That is most likely the best place to put something like this. It is essentially a set of notebook tutorials for the packages.